spec/views/layouts/application_spec.rb

```diff
@@ -1,27 +0,0 @@
-RSpec.describe "layouts/application" do
-  # standard:disable Lint/ConstantDefinitionInBlock
-  class ActionView::TestCase::TestController
-    include Auth
-
-    def current_user
-      nil
-    end
-  end
-  # standard:enable Lint/ConstantDefinitionInBlock
-
-  it "shows the meta tags when ROBOT_NOINDEX is set to true" do
-    ClimateControl.modify ROBOT_NOINDEX: "true" do
-      render
-      expect(response).to include("<meta content='noindex' name='robots'>")
-      expect(response).to include("<meta content='noindex' name='googlebot'>")
-    end
-  end
-
-  it "does not show the meta tags when ROBOT_NOINDEX is set to false" do
-    ClimateControl.modify ROBOT_NOINDEX: "false" do
-      render
-      expect(response).to_not include("<meta content='noindex' name='robots'>")
-      expect(response).to_not include("<meta content='noindex' name='googlebot'>")
-    end
-  end
-end
```

Comment:
And what's the reason for this removal?

Reply:
Good question. I will take another look, I'd somehow missed this

Reply:
Hi Ynda - I've now added this file back in. It looks from the git logs like it was deleted when the new ruby version was installed, however I've not been able to nail down exactly why that was. It was last amended two years ago to stop a previous Ruby update from breaking tests with some new style requirements. 

Tests pass both with and without it, so it appears to make no difference, however I would rather not delete a spec file for no good reason.